cms/static/js/views/xblock_string_field_editor.js

```diff
@@ -30,9 +31,10 @@ define(['js/views/baseview', 'js/views/utils/xblock_utils'],
 
             render: function() {
                 this.$el.append(this.template({
-                    value: this.model.escape(this.fieldName),
+                    value: this.model.get(this.fieldName),
```

Comment:
The escape was removed here, which is correct. But then the display name must be escaped in the template.